scripts/fast-reboot

```diff
@@ -477,7 +477,7 @@ debug "Stopped  syncd ..."
 # then immediately call `systemctl stop ...` to prevent the service from
 # restarting the container automatically.
 for CONTAINER_NAME in $(docker ps --format '{{.Names}}'); do
-    docker kill $CONTAINER_NAME > /dev/null
+    docker kill $CONTAINER_NAME > /dev/null || [ $? == 1 ]
```

Comment:
Per our discussion, it might be best to continue on all non-zero exit codes here, preferably logging them to syslog for posterity.

Reply:
Updated the change.